error-handling.js

```diff
@@ -1,7 +1,26 @@
+const { selectCategories, selectReviews,selectReviewsById } = require("./models/review_id");
+
 const handle404paths = (req, res, next) => {
     res.status(404).send({ msg: 'path not found!' })
 };
 
+const handleSpecificErrors = (err, req, res, next) => {
+    if (err.code ==='22P02') {
+        res.status(400).send({ msg: 'path not found!'});
+    } else {
+        next(err);
+    }
+}
+
+const handle500s = (err, req, res, next) =>{
+    console.log(err);
```

Comment:
Don't need this console log here as you're dealing with the actual error by returning it to the user as a status 500 with a message, so this console log can be removed.